notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_2_0.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>see "Known issue" section: <a href="https://screenshot.googleplex.com/93dniR5gC2khMqw" target="_blank">https://screenshot.googleplex.com/93dniR5gC2khMqw</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Do you suggest to add the known issues to the notebook?</p>

Reply:
<p>I posted the question in our chat group with Sam, if we are gonna add it to notebooks please add to both, otherwise probably Sam only need to add it on public doc.</p>

Reply:
Added